ecosystem/sep-0031.md

```diff
@@ -329,32 +317,7 @@ The response should be a JSON object like:
       "fee_fixed": 5,
       "fee_percent": 1,
       "min_amount": 0.1,
-      "max_amount": 1000,
-      "sep12": {
-        "sender": {
-          "types": {
-            "sep31-sender": {
-              "description": "U.S. citizens limited to sending payments of less than $10,000 in value"
-            },
-            "sep31-large-sender": {
-              "description": "U.S. citizens that do not have sending limits"
-            },
-            "sep31-foreign-sender": {
-              "description": "non-U.S. citizens sending payments of less than $10,000 in value"
-            }
-          }
-        },
-        "receiver": {
-          "types": {
-            "sep31-receiver": {
-              "description": "U.S. citizens receiving USD"
-            },
-            "sep31-foreign-receiver": {
-              "description": "non-U.S. citizens receiving USD"
-            }
-          }
-        }
-      }
+      "max_amount": 1000
```

Comment:
I think it's possible to use the new KYC flow and keep the `sep12` block right? The new flow works if the SEP-12 request is `type=sep31-large-sender` and `transaction_id=...`. The special types `sep31-sender` and `sep31-receiver` can be hardcoded values we use in the Anchor Platform but we don't need to restrict them at the protocol level. Otherwise, this would be a breaking change for SEP-31 clients.

Reply:
It's possible, but I don't think it's necessary because the sending anchor no longer needs to classify customers.

In the new flow, sending anchors can pre-register the sender and receiver with some basic information. Then, it’s up to the receiving anchor to determine what additional information is needed (such as whether the customer is foreign or a large sender) based on the transaction details and return the required fields to the sending anchor. This way, the receiving anchor can manage its own classified KYC customer decision-making process.

What are your thoughts?

Additionally, since SEP-31 is not currently being adopted by any clients, it's a good opportunity to make these improvements.

Reply:
I don't mind getting rid of custom-define types entirely, but we would need to deprecate it, rather than straight removing

Reply:
I'm also worry about some gaps in this flow with customized customer type.

If the sending anchor create a SEP-31 transaction without pre-register the customers, and is doing kyc now, how would they know what customer `type` they should use in the `GET /cutomer?transaction_id=<>&type=<> ` call?

Reply:
It's up for the anchor to decide, generally. Client is only supposed to either use `sep-6` or `sep-31` when interacting with business that uses AP